anvil/src/shell.rs

```diff
@@ -168,6 +168,11 @@ impl PointerGrab for ResizeSurfaceGrab {
         _serial: Serial,
         _time: u32,
     ) {
+        // It is impossible to get `min_size` and `max_size` of dead toplevel, so we return early.
+        if !self.toplevel.alive() | self.toplevel.get_surface().is_none() {
+            return;
```

Comment:
I think we'd need to unset the the grab as well as doing an early return. The client exiting should void any pending grab.

Reply:
Indeed, my bad.